src/models-api.ts

```diff
@@ -33,16 +31,8 @@ export type ModelSchemaParameter = {
 };
 
 export class ModelsAPI {
-  inference: OpenAI;
   private _models: Model[] | null = null;
 
-  constructor(apiKey: string) {
-    this.inference = new OpenAI({
-      baseURL: "https://models.inference.ai.azure.com",
-      apiKey,
-    });
```

Comment:
I moved setting the endpoint URL into the `prompt` call in `index.ts`. But we could keep an instance property, I'd just need to [implement `prompt.defaults()`](https://github.com/copilot-extensions/preview-sdk.js/issues/52) first